src/pages/home/Home.js

```diff
@@ -1,28 +1,54 @@
 import React from 'react';
-import 'i18n/i18n';
 import { translate } from 'react-i18next';
+import 'i18n/i18n';
+
+import styles from './Home.scss';
 import { Navbar } from 'components';
-import styles from './assets/home.scss';
-import meeting from './assets/meeting.svg';
+import HomeSection from './components/HomeSection';
+import ImageAbout from './assets/images/about.svg';
+import ImageMission from './assets/images/mission.svg';
+import Contact from 'components/contact/Contact';
 
-const Home = ({ t }) => {
+const heroImage = require('./assets/images/meeting.svg');
+
+function Home({ t }) {
   return (
-    <section className={styles.homeContainer}>
-      <div className={styles.mainHeaderBackground} />
+    <React.Fragment>
       <Navbar />
-      <div className={styles.mainHeaderContent}>
-        <div>
-          <h1 className={styles.header}>{t('Coding')}</h1>
-          <h1 className={styles.header}>{t('Coach')}</h1>
-          <h4>{t('Connecting developers with mentors worldwide')}</h4>
-          <button className={styles.learnMore}>{t('Learn More')}</button>
-        </div>
-        <div>
-          <img alt="coding coach logo" src={meeting} />
+      <section className={styles.hero}>
+        <div className={styles.heroInner}>
+          <header>
+            <h1 className={styles.heroTitle}>
+              <span>{t('Coding')}</span>
+              <br />
+              <span>{t('Coach')}</span>
+            </h1>
+            <h2 className={styles.heroSubtitle}>
+              {t('Connecting developers with mentors worldwide')}
+            </h2>
+            <button className={styles.callToAction}>{t('Learn More')}</button>
+          </header>
+          <img className={styles.heroImage} alt="coding coach logo" src={heroImage} />
```

Comment:
I think we should use i18n for this static alt text also, just like we have done in all other places.

Reply:
I actually translated them... They have been overwritten by a merge or something!
**Good point** :+1: